test/mocha/40-database.js

```diff
@@ -123,11 +124,16 @@ describe('ZCaps Database Tests', () => {
         referenceId: mockData.zcaps.alpha.referenceId,
         capability: mockData.zcaps.alpha.capability
       });
-      await brZcapStorage.zcaps.insert({
-        controller: mockData.zcaps.beta.controller,
-        referenceId: mockData.zcaps.beta.referenceId,
-        capability: mockData.zcaps.beta.capability
-      });
+      for(let i = 0; i < 1000; i++) {
+        await brZcapStorage.zcaps.insert({
+          controller: uuid(),
```

Comment:
Note: This is going to create a different controller every time -- which is less likely to trigger the proper index. We want these records to match the controller that we're running the index tests on. The idea here is to make it so there are many matches for a given controller on the `{controller: 1, referenceId: 1}` index ... so that it's inefficient for that to be picked over the `{controller: 1, id: 1}` index when `id` is given in the query.